.vscode/launch.json

```diff
@@ -0,0 +1,52 @@
+{
```

Comment:
I don't think we want to include these files in the repo's .vscode as they are related to ToDo app.

Reply:
I added them for an easier local dev debug experience. but can remove them